src/sass/components/_plugins.scss

```diff
@@ -3,6 +3,21 @@
 // https://www.redmine.org/plugins/additionals
 // --------------------------------------------------
 
+#sidebar .dashboards .dashboard-system-default {
+  position: absolute;
+  transform: translateY(-175%);
+  @if $sidebar-position == "right" {
+    right: 15px;
+  }
+}
+
+.flyout-menu .dashboard-system-default {
+  position: absolute;
+  right: 10px;
+  transform: translateY(-190%);
```

Comment:
This (and -175% above) are a bit specific/odd values. Do they work correctly on different window sizes? If yes - can you add a comment and explain why these values?

Reply:
Yes this works on different windows sizes, but it doesn't when changing some variables.

The icon added by the plugin is outside the `a` tag used to draw the tab. So I solved this by computing existing variables. It should be more explicit.